tasklist/client/e2e/docs/docs-snapshots.spec.ts

```diff
@@ -0,0 +1,521 @@
+      tenantId: '<default>',
+      bpmnXml: null,
+    }),
+  );
+  await page.route(/^.*\/v1\/internal\/processes$/i, (route) => {
+    return route.fulfill({
+      status: 200,
+      body: JSON.stringify(processes),
+      headers: {
+        'content-type': 'application/json',
+      },
+    });
+  });
+  await page.route(/^.*\/v1\/internal\/processes\?.*$/i, (route, req) => {
+    const params = new URLSearchParams(req.url().split('?')[1]);
+    const query = params.get('query');
+    return route.fulfill({
+      status: 200,
+      body: JSON.stringify(
+        query
+          ? processes.filter(
+              (p) => p.name.includes(query) || p.bpmnProcessId.includes(query),
+            )
+          : processes,
+      ),
+      headers: {
+        'content-type': 'application/json',
+      },
+    });
+  });
+}
+
+async function mockProcessStart(page: Page) {
+  await page.route(/^.*\/v1\/internal\/processes\/[^/]*\/start$/i, (route) => {
+    return route.fulfill({
+      status: 200,
+      body: JSON.stringify({id: '1234'}),
+      headers: {
+        'content-type': 'application/json',
+      },
+    });
+  });
+}
+
+async function mockCurrentUser(page: Page) {
+  await page.route(/^.*\/v1\/internal\/users\/current$/i, (route) =>
+    route.fulfill({
+      status: 200,
+      body: JSON.stringify({
+        userId: 'demo',
+        displayName: 'demo',
+        permissions: ['READ', 'WRITE'],
+        salesPlanType: null,
+        roles: null,
+        c8Links: [],
+        tenants: [],
+      }),
+      headers: {
+        'content-type': 'application/json',
+      },
+    }),
+  );
+}
+
+async function mockClientConfig(page: Page) {
+  await page.route('**/client-config.js', (route) =>
+    route.fulfill({
+      status: 200,
+      headers: {
+        'Content-Type': 'text/javascript;charset=UTF-8',
+      },
+      body: `window.clientConfig = {
+        "isEnterprise":true,
+        "canLogout":true,
+        "isLoginDelegated":false,
+        "contextPath":"",
+        "organizationId":null,
+        "clusterId":null,
+        "stage":null,
+        "mixpanelToken":null,
+        "mixpanelAPIHost":null,
+        "isMultiTenancyEnabled": false
+      };`,
+    }),
+  );
+}
+
+test.beforeEach(async ({page}) => {
+  await mockCurrentUser(page);
+  await mockClientConfig(page);
+  await page.addInitScript(() => {
+    window.localStorage.setItem('hasConsentedToStartProcess', 'true');
+  });
+});
+
+test.describe('Tasklist snapshots', () => {
+  test('empty tasklist', async ({page, taskPanelPage}) => {
+    await mockTaskSearch(page, []);
+
+    await taskPanelPage.goto();
+
+    await expect(page.getByTestId('tasks-skeleton')).not.toBeVisible();
+
+    await expect(page).toHaveScreenshot();
+  });
+
+  test('populated tasklist', async ({page, taskPanelPage}) => {
+    await mockTaskSearch(page, MOCK_TASKLIST);
+
+    await taskPanelPage.goto();
+
+    await expect(page.getByTestId('tasks-skeleton')).not.toBeVisible();
```

Comment:
I think you can also just wait for the task content instead of waiting for the tasks, it should auto-wait. It's not like in the unit tests